src/config.rs

```diff
@@ -135,8 +159,8 @@ impl Config {
 #[serde(default)]
 pub struct Public {
     /// Public address.
-    #[serde(deserialize_with = "to_socket_addrs")]
-    pub address: SocketAddr,
+    // #[serde(deserialize_with = "to_socket_addrs")]
+    pub address: String, // SocketAddr,
```

Comment:
If we keep this for the bind, this should be put back.

Reply:
Thanks, this was put back